CONTRIBUTING.md

```diff
@@ -82,6 +82,27 @@ some bugs or smaller features.  We have a [feature development
 process](https://github.com/kubernetes/features/blob/master/README.md), but
 navigating the Kubernetes system as a newcomer can be very challenging.
 
+### Building the project
+
+There are a few things you need to build and test Kubernetes:
+1. `make` - the human interface to the Kubernetes build is `make`, so you must
+   have this tool installed on your machine.  We try not to use too many crazy
+   features of `Makefile`s and other tools, so most commonly available versions
+   should work.
+1. `docker` - some parts of the build/test system depend on `docker`.  You
+   need a relatively recent version of Docker installed, and available to you.
```

Comment:
+1 for Ivan, minikube is *not* for developers, for developers look at https://github.com/kubernetes/kubernetes/pull/36590#issuecomment-259802736 or https://github.com/kubernetes/kubernetes/pull/36590#issuecomment-259808686